configurer/linux/coreos.go

```diff
@@ -0,0 +1,35 @@
+package linux
+
+import (
+	"github.com/k0sproject/rig"
+	"github.com/k0sproject/rig/os"
+	"github.com/k0sproject/rig/os/registry"
+	"strings"
+)
+
+
+// coreos provides OS support for Fedora & RHEL core os systems
+type Coreos struct {
+	os.Linux
+	BaseLinux
+
+}
+
+func init() {
+	registry.RegisterOSModule(
+		func(os rig.OSVersion) bool {
+			return os.ID == "fedora" && strings.Contains(os.Name, "CoreOS") || os.ID == "rhel" && strings.Contains(os.Name, "CoreOS")
+		},
+		func() interface{} {
+			return &Coreos{}
+		},
+	)
+}
+
+func (l Coreos) InstallPackage(h os.Host, pkg ...string) error {
+	return h.Execf("sudo rpm-ostree install %s --reboot", strings.Join(pkg, " "))
+}
+// InstallKubectl installs kubectl using the rpm-ostree. not reliable now as we need to figure out reconnection to the hosts.  
+//func (l Coreos) InstallKubectl(h os.Host) error {
+//	return l.InstallPackage(h, "kubectl")
+//}
```

Comment:
There's kubectl already embedded inside k0s and that is used in k0sctl for all `kubectl`ing, so no need to install it separately.

Reply:
I suppose we can drop the `InstallPackage` func as well and rely on the linux default?